pkg/logs/tailers/journald/tailer.go

```diff
@@ -55,10 +55,9 @@ type Tailer struct {
 // NewTailer returns a new tailer.
 func NewTailer(source *sources.LogSource, outputChan chan *message.Message, journal Journal, processRawMessage bool) *Tailer {
 	if len(source.Config.ProcessingRules) > 0 && processRawMessage {
-		log.Warn("Log processing rules with the journald collection will change in a future version of the Agent:")
-		log.Warn("The processing will soon apply on the message content only instead of on the structured log (e.g. on the collected JSON).")
-		log.Warn("In order to immediately switch to this new behaviour, set 'process_raw_message' to 'false' in your logs integration config.")
-		log.Warn("Please reach Datadog support if you have more questions.")
+		log.Warn("The logs processing rules can now be applied to the message content only instead of the whole JSON-structured log (e.g. on the collected JSON). This behavior will be the default behavior in a future version of the Agent.")
```

Comment:
```suggestion
		log.Warn("The logs processing rules currently apply to the raw JSON-structured log, which may be unintuitive for journald logs. These rules can now be applied to the message content only, and this behavior will be the default behavior in a future version of the Agent.")
```
My thought here is that the warning log should make it obvious what the current mode is.

Reply:
Good suggestion. WDYT if I remove the `which may be unintuitive for journald logs` from your suggestion? I don't think it's worth justifying the change here.
Leading to:
`The logs processing rules currently apply to the raw JSON-structured log. These rules can now be applied to the message content only, and this behavior will be the default behavior in a future version of the Agent.`

Reply:
Yeah, good to go with that wording 👍